### PR TITLE
Use vscode services

### PR DESCRIPTION
**Pull request by CGNonofr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "types": "dist/index.d.ts",
   "dependencies": {
     "monaco-editor": "npm:@codingame/monaco-editor@^0.33.6",
+    "vscode": "npm:@codingame/monaco-vscode-api@^1.67.9",
```
Comment (CGNonofr) [inline package.json:70]: should use ~ instead

**Review by samuel-olivier — APPROVED** [review]
Nice !

File: package.json (hunk 1 above)
Comment (CGNonofr) [reply package.json]: 8c0dcba280813490a5d016f80cbf6687e8785bb3

**Comment by CGNonofr** [comment]
@samuel-olivier 8c0dcba280813490a5d016f80cbf6687e8785bb3

**Review by samuel-olivier — APPROVED** [review]

**Pull request merged**